src/bicycle/model.py

```diff
@@ -37,14 +121,60 @@ def __init__(self, x_dim: int, z_dim: int, n_cond: int, act_fn: object = nn.GELU
         self.net.apply(init_weights)
 
     def forward(self, x):
+        """Overrides pyTorchs nn.Module.forward function."""
         x = self.net(x)
         mu = x[:, : self.z_dim]
         variance = torch.nn.Softplus()(x[:, self.z_dim :]) + 1e-6
         return mu, variance
 
 
-class Omega_Iterative(pl.LightningModule):
-    def __init__(self, alpha, beta, sigma, lr=1e-2, sigma_min=1e-3, device="cpu"):
+class OmegaIterative(pl.LightningModule):
+    """
+    This class optimizes omega to fit the latent normal distribution of z.
```

Comment:
No, it does not optimize omega. It gives a parametrization for omega and computes B.